core/src/apps/common/writers.py

```diff
@@ -1,4 +1,4 @@
-from trezor.utils import ensure
+from trezor.utils import empty_bytearray, ensure  # noqa: F401
```

Comment:
I think `empty_bytearray` is not needed anymore? Maybe it was before I rebased.

Reply:
You are right, I removed it and the tests are passing just fine.